.github/workflows/cleanup-dp.yml

```
@@ -0,0 +1,31 @@
+name: Cleanup Deploy Previews
+
+on:
+  # This line enables manual triggering of this workflow.
+  schedule:
+    - cron: 0 0 * * *
+
+jobs:
+  execute:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the code
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install relevant packages
+        run: |
+          which aws
+          sudo apt update -q && sudo apt install -y curl unzip less jq 
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.23.6/bin/linux/amd64/kubectl && \
+          sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl && \
+          curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 && \
+          chmod 700 get_helm.sh; ./get_helm.sh
```

It's a little hard to read when some lines have multiple commands, and some don't. If it's hard to read, it'll be hard to debug next year. 🙂

```suggestion
          sudo apt update -q
          sudo apt install -y curl unzip less jq 
          curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.23.6/bin/linux/amd64/kubectl
          sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
          curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
          chmod 700 get_helm.sh
          ./get_helm.sh
```